llvm/test/CodeGen/AArch64/avg.ll

```diff
@@ -17,6 +17,28 @@ define <16 x i16> @zext_avgflooru(<16 x i8> %a0, <16 x i8> %a1) {
   ret <16 x i16> %avg
 }
 
+define void @zext_mload_avgflooru(ptr %p1, ptr %p2, <vscale x 8 x i1> %mask) {
```

Comment:
This feels like it should be in a sve file, like sve-hadd.ll

Reply:
Done!